packages/formation/sass/modules/_m-button.scss

```diff
@@ -7,6 +7,8 @@ button,
 [type='submit'],
 [type='reset'],
 [type='image'] {
+  -webkit-font-smoothing: inherit;
+  -moz-osx-font-smoothing: inherit;
```

Comment:
I've update this to `inherit` because this seems to be what the USWDS v3 system uses for their `no-knockout-font-smoothing` mixin:

https://github.com/uswds/uswds/blob/1db0e68d7e5cf21ae52f2d8a29d5963c7aa1b681/packages/uswds-core/src/styles/mixins/general/add-knockout-font-smoothing.scss